README.md

```diff
@@ -140,6 +140,31 @@ quark -a first.apk -a second.apk -C
 
 <img src="https://i.imgur.com/ClRWOei.png"/>
 
+### Radiocontrast
+The Radiocontrast is a Quark API that can quickly generate 100% confidence rules with the Native APIs used in a specific method. The feature lets you easily see through the behavior of a method, just like radiocontrast.
```

Comment:
+ I suggest removing the article `the` and the word `can` since they are unnecessary.

+ I suggest splitting the first statement to make the idea easy to understand. For example, 

  ```
  Radiocontrast is a Quark API that quickly generate 100% confidence rules with the Native APIs used in a specific method.
  ->
  Radiocontrast is a Quark API that quickly generates Quark rules from a specified method. It builds up 100% matched rules by using native APIs in that method.
  ```

+ I suggest using the word "expose" since its meaning is closer to what we want to deliver in the last sentence.